drivers/bt_ll_nrfxlib/clock_control/Kconfig

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config CLOCK_CONTROL_NRF_LL_NRFXLIB
+	bool "Nordic BLE controller clock control driver"
```

Comment:
We need consistent naming - is it nRF BLE Controller or Nordic BLE controller?

Reply:
fixed